index.html

```diff
@@ -51,7 +51,115 @@ <h3>I'm a Software Developer</h3>
             <li><a href="https://www.linkedin.com/in/philip-modinos-14195021"><i class="fa fa-linkedin-square fa-3x" aria-hidden="true"></i></a></li>
         </ul>
     </aside>   
+    
+</section> <!--page 1 end -->
 
-</div>
+<section class="page2">
+    <div class="container">
+        <div class="items title1">
+           <h3>My Recent Works</h3>
+        <br> <br>
+        <hr>
+        </div>
+        <div class="items img1">
+            <img src="images/testimg.png" alt="">
+        </div>
+        <div class="items description1">
+            <h3 class="title2">Multi-Post Stories</h3>
+            <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
```

Comment:
[OPTIONAL] I think you need to stick to the design template as much as possible. There are some discrepancies between your Design and that of Figma and it will enhance the quality of your portfolio to get them fixed.
## Your Design:point_down:
![image](https://user-images.githubusercontent.com/45668340/148454981-a4ff957a-0fe8-411a-853e-f32914422ebf.png)

## Figma Design:point_down:
![image](https://user-images.githubusercontent.com/45668340/148455049-bb999e84-12d0-4d67-92cd-92781bc2a501.png)